xds/internal/xdsclient/xdsresource/unmarshal_cds.go

```diff
@@ -81,6 +82,28 @@ const (
 )
 
 func validateClusterAndConstructClusterUpdate(cluster *v3clusterpb.Cluster) (ClusterUpdate, error) {
+	stringMD := make(map[string]string)
+	if fmd := cluster.GetMetadata().GetFilterMetadata(); fmd != nil {
+		if val, ok := fmd["com.google.csm.telemetry_labels"]; ok {
+			/*
+				"service_name" = "",
+				"service_namespace" = ""
+			*/
```

Comment:
This comment looks out of place.  Are you documenting JSON schema or.......?

Reply:
Deleted.